exercises/concept/ghost-gobble-arcade-game/arcade_game_test.py

```diff
@@ -100,3 +100,11 @@ def test_win_if_all_dots_eaten_and_touching_a_ghost_with_a_power_pellet_active(s
             True,
             msg="win if all dots eaten and touching a ghost with a power pellet active"
         )
+
+    @pytest.mark.task(taskno=4)
+    def test_dont_win_if_not_all_dots_eaten(self):
```

Comment:
I tested this code both locally and on the `python-test-runner`, it works completely OK. Exemplar passes the tests and the previous `WRONG Example` is now failing, which is a good thing in this case.